widgetssdk/src/main/AndroidManifest.xml

```diff
@@ -87,7 +87,9 @@
             android:name=".core.screensharing.MediaProjectionService"
             android:foregroundServiceType="mediaProjection" />
 
-        <service android:name=".core.notification.NotificationRemovalService" />
+        <service
+            android:name=".core.notification.NotificationRemovalService"
+            android:exported="false" />
```

Comment:
It is impossible to leave proper comments in XML files, but can you just write here for me why `exported="false"` is needed to solve this issue?